pages/about/groups/task-forces/wcag2ict/work-statement.md

```diff
@@ -24,113 +24,127 @@ The WCAG2ICT Task Force is a task force of the [Accessibility Guidelines Working
 
 ## Status
 
-This work statement was [approved by the AG WG on 26 October 2022](https://lists.w3.org/Archives/Public/w3c-wai-gl/2022AprJun/0280.html). The task force is active.
+This work statement was [approved by the AG WG on @@insert date approved and update URL@@](https://lists.w3.org/Archives/Public/w3c-wai-gl/2022AprJun/0280.html). The task force is active.
```

Comment:
@daniel-montalvo The approval date and related email URL need update.

Reply:
```suggestion
This work statement was approved by the AG Chairs WG on 24 January 2024. The task force is active.
```

WG approval is no longer required. Chairs approve Work Statements. Updated accordingly.

Reply:
@daniel-montalvo @maryjom I am reopening this conversation since the current version still states the work statement was approved on 24 January 2024. Isn't that a typo?

Reply:
@remibetin I have just committed the change to 2025. Thanks for catching this.